README.md

```diff
@@ -53,45 +48,33 @@ git clone https://github.com/NetworkGradeLinux/meta-mion-backports.git
 ## Basic Usage
 
 To begin, set up the build environment using the OpenEmbedded init script.
-Afterwords you can use our build script. Running `../mc_build.sh` without
+Afterwords you can use our build script. Running `../cronie.sh` without
 arguments displays basic usage. **In general:**
 
 ```shell
 source openembedded-core/oe-init-build-env
-../mc_build.sh -m <machine> -c <container config>:<container image> -h <host config>:<host_image> -d container_image
+../cronie.sh -m <machine> <image>
 ```
 
 To do a "dry run" without running a build, add `-e` which emits what would have
 run if you ran this from bitbake.
-If you want to disable the autostarting of the container, use -d with a comma
-delineated list of the container image name.
 
 If you are familiar with Yocto Project development and multiconfig and wish to
 use bitbake directly, see the local.conf for variables that need to be set. For
 running bitbake commands, including outside of building, such as
-`bitbake-layers show-layers`, run `../mc_build.sh` as you otherwise would with
+`bitbake-layers show-layers`, run `../cronie.sh` as you otherwise would with
 the `-e` flag from the build directory and prepend the `BB_ENV_EXTRAWHITE`
 output to the bitbake command.
 
 Example:
 
 ```shell
-BB_ENV_EXTRAWHITE="ALL_PROXY BBPATH_EXTRA BB_LOGCONFIG BB_NO_NETWORK BB_NUMBER_THREADS BB_SETSCENE_ENFORCE BB_SRCREV_POLICY DISTRO FTPS_PROXY FTP_PROXY GIT_PROXY_COMMAND HTTPS_PROXY HTTP_PROXY MACHINE NO_PROXY PARALLEL_MAKE SCREENDIR SDKMACHINE SOCKS5_PASSWD SOCKS5_USER SSH_AGENT_PID SSH_AUTH_SOCK STAMPS_DIR TCLIBC TCMODE all_proxy ftp_proxy ftps_proxy http_proxy https_proxy no_proxy  BUILD_ARGS CONTAINER_NAMES CONTAINER_DEPENDS BBMULTICONFIG MION_CONT_DISABLE VENDOR" BUILD_ARGS=" host-onie:mion-image-onlpv1" BBMULTICONFIG=" host-onie" CONTAINER_NAMES="" CONTAINER_DEPENDS="" VENDOR="stordis" MACHINE="stordis-bf2556x-1t" MION_CONT_DISABLE="" bitbake-layers show-layers
+BB_ENV_EXTRAWHITE="ALL_PROXY BBPATH_EXTRA BB_LOGCONFIG BB_NO_NETWORK BB_NUMBER_THREADS BB_SETSCENE_ENFORCE BB_SRCREV_POLICY DISTRO FTPS_PROXY FTP_PROXY GIT_PROXY_COMMAND HTTPS_PROXY HTTP_PROXY MACHINE NO_PROXY PARALLEL_MAKE SCREENDIR SDKMACHINE SOCKS5_PASSWD SOCKS5_USER SSH_AGENT_PID SSH_AUTH_SOCK STAMPS_DIR TCLIBC TCMODE all_proxy ftp_proxy ftps_proxy http_proxy https_proxy no_proxy  VENDOR" VENDOR="stordis" MACHINE="stordis-bf6064x-t" bitbake  "mion-onie-image-onlpv1"
 ```
 
 ## Examples
```

Comment:
We may want to just change that to "Example", or, perhaps add using it for another machine, such as qemu, but that's not a big enough reason to not merge.